tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -114,10 +120,12 @@ def test_non_ocs_taint_and_tolerations(self):
             )
         else:
             param = (
-                f'{{"spec": {{"placement": {{"all": {tolerations}, "mds": {tolerations}, '
-                f'"noobaa-core": {tolerations}, "rgw": {tolerations}}}}}}}'
+                f'"all": {tolerations}, "mds": {tolerations}, '
+                f'"noobaa-core": {tolerations}, "rgw": {tolerations}, '
             )
-
+            if version.get_semantic_ocs_version_from_config() >= version.VERSION_4_15:
```

Comment:
Most of the steps in this test will have to be modified according to the new 4.16 custom taint features. How do you plan to implement the new feature changes to this test?

Reply:
Resolving this comment, based on the offline discussion with Vishakha